### DISCUSSION TITLE
Reducing the bus factor + reorg

### DISCUSSION THREAD

**Discussion by inverse** [web-scrobbler/web-scrobbler#3140] [General]
@web-scrobbler/web-scrobbler-team  what do you think of this proposal for reducing the bus factor.

Right now @alexesprit holds the majority of access which would be extremely difficult to recover from for example for things like

- Rotating Chrome/Firefox tokens for pushing to the stores
- Publishing to Edge - still manual due to the lack of API on Microsoft side.

I propose we leverage some team password manager to share these. [1Password](https://github.com/1Password/1password-teams-open-source) are even offering it free for Open Source software.

Along with this @alexesprit is the one in control of the web-scrobbler domain name. If we would adopt something like [OpenCollective](https://opencollective.com/) I feel this would lend it self to a more community driven way in which we can expense such things.

What are your thoughts on such a transition?

**Comment by alexesprit** [comment]
Not familiar with 1Password. If you can create an account in few clicks, let's do this. I think we can use the project email (web.scrobbler@gmail.com) to register (if they ask email of course). Then I'll add credentials to project related things (Twitter, domain management etc)?

Not sure about how to solvethe MS Edge update question - not sure if they provide multiuser access. Probably we could create a new MS account using the project's email, and move the MS extension project somehow to new account.

**Reply by alexesprit** [reply]
As I mentioned in Discord DM we already have OpenCollective account, so I'll just need to figure out if we can share the access across project core members.

**Reply by alexesprit** [reply]
@inverse if you have an OpenCollective account, could you share account name (or email), and I'll add you to the project team list on OpenCollective. If no, I can invite you (what email should I use btw?).

**Reply by inverse** [reply]
@alexesprit I've pinged you my email via Discord

**Reply by inverse** [reply]
@alexesprit for 1Password we would need to follow this: https://github.com/1Password/1password-teams-open-source

If you don't think we need to share credentials then we can drop this.. I think we covered most cases.

Using 1Password would allow us to have like a shared vault that we could authenticate with our own accounts.

**Comment by alexesprit** [comment]
> Rotating Chrome/Firefox tokens for pushing to the stores

1. Firefox. You can obtain credentials here - https://addons.mozilla.org/en-US/developers/addon/api/key/. Just log in using your Mozilla account and you'll see them. I'm pretty sure you can use yours as well, as your Mozilla account is linked to the extension.
2. Chrome. It's difficult, as I have no admin rights here :( we should ping David to add you or other people.

**Reply by inverse** [reply]
@david-sabata would you be able to grant more users access to chrome or make @alexesprit  admin?

**Reply by david-sabata** [reply]
Hi! I just upgraded you to a manager of the publishing group @alexesprit. You should now be able to add more people at https://groups.google.com/g/web-scrobbler/members

**Reply by inverse** [reply]
Can confirm I can generate tokens for Firefox.

**Reply by alexesprit** [reply]
Ok cool. You can also regenerate tokens for Chrome Web Store using this guide: https://github.com/fregante/chrome-webstore-upload/blob/main/How%20to%20generate%20Google%20API%20keys.md. I made you an owner the project org in Google Cloud Platform where we manage API for autopublishing the extension.

https://console.cloud.google.com/apis/api/chromewebstore.googleapis.com/credentials?authuser=0&project=web-scrobbler-team Publish script

**Reply by inverse** [reply]
![image](https://user-images.githubusercontent.com/713525/164401597-415a894b-ea3d-4ad2-9819-15e1dfa3c94c.png)

@alexesprit I'm missing some permissions it seems

**Reply by alexesprit** [reply]
Hmm, I thought you are an owner of the project on GCP. I'll review and update your permissions if neccessary.

**Reply by inverse** [reply]
@alexesprit ignore me  - I was in the wrong Google account :/
